### ISSUE TITLE
Signed token no longer includes payload.iss

### ISSUE THREAD

**Issue by jonekdahl** [auth0/node-jsonwebtoken#196]
When providing an issuer through an object payload to `jwt.sign` (via `payload.iss`) it is no longer included in the signed token. Neither is `aud`, `sub` or `jti`.

Test case (breaks with 6.1.0, works with 5.7.0):
```javascript
  it('should use issuer provided in payload.iss', function () {
    var token = jwt.sign({ iss: 'foo' }, 'shhhhh');
    var decoded_issuer = JSON.parse(b64_to_utf8(token.split('.')[1])).iss;
    expect(decoded_issuer).to.equal('foo');
  });
```

Result (with 6.1.0):
```
AssertionError: expected undefined to equal 'foo'
```

**Comment by holm** [comment]
Also ran into problem after upgrading to the latest version

**Comment by jfromaniello** [comment]
Thanks for reporting this and for the test case. I have fixed it on `v6.1.1`.
